### PR TITLE
Bug DCOS-9371: Makes named VIP subnet configurable from gen

### PR DESCRIPTION
**Pull request by GoelDeepak**
Changes /etc/minuteman.app.config to
 - /etc/minuteman.config.d/master.config,
 - /etc/minuteman.config.d/common.config

### CODE REVIEW

File: src/minuteman_app.erl [hunk 1]
```diff
@@ -5,27 +5,43 @@
 %% Application callbacks
 -export([start/2, stop/1]).
 
--define(DEFAULT_CONFIG_LOCATION, "/opt/mesosphere/etc/minuteman.app.config").
+-define(DEFAULT_CONFIG_LOCATION, "/opt/mesosphere/etc/minuteman.config.d").
```
Comment (sargun) [inline src/minuteman_app.erl]: probably makes sense to rename this?

Comment (sargun) [inline src/minuteman_app.erl]: Maybe DEFAULT_CONFIG_DIRECTORY?

**Comment by sargun** [comment]
Automated style checks didn't pass. Please fix.

**Comment by GoelDeepak** [comment]
which one are you referring here? Is it the one that was running
under dcos/dcos PR? If yes, then that is a vagrant test which fails 50% of
the times. It depends on whether your job was the first one to run or
somebody else. I will re-run it and it will pass then.

File: src/minuteman_app.erl (hunk 1 above)
Comment (GoelDeepak) [inline src/minuteman_app.erl]: Done.

**Pull request merged**